node/src/cli.rs

```diff
@@ -75,7 +75,7 @@ pub struct BuildSpecCmd {
     #[arg(long, value_name = "TYPE", value_parser = parse_chaintype, default_value = "live")]
     pub chain_type: ChainType,
 
-    #[arg(long, default_value = "4_000_000")]
```

Comment:
I think only this change is necessary. The others should be okay. Thank you for finding this.

Reply:
Let me correct and check

Reply:
Checked - you are right. I reverted the changes I did